bindings/nostr-sdk-ffi/src/protocol/event/builder.rs

```diff
@@ -166,6 +166,12 @@ impl EventBuilder {
 
     /// Text note reply
     ///
+    /// This adds only that most significant tags, like:
```

Comment:
```suggestion
    /// This adds only the most significant tags, like:
```

Same typo in the other docs